runtime/examples/run.cpp

```diff
@@ -23,19 +23,15 @@ limitations under the License.
 using namespace std;
 
 // TODO: delete this func. it is for debug. 
-blueoil::Tensor RandomImage(int num, int height, int width, int channel) {
-    blueoil::Tensor t;
-    const int data_size = channel * width * height * num;
-    vector<float> data(data_size, 0);
-    vector<int> shape = {num, height, width, channel};
+blueoil::Tensor RandomImage(int height, int width, int channel) {
+    blueoil::Tensor t({height, width, channel});
+    vector<float> data = t.data();
```

Comment:
This will create a copy of the vector.
So below loop will update it.
And as you return `t` after all you will still return it with unchanged `data`...

Maybe you were going to do something like:
```cpp
vector<float>& data = t.data();
```
instead...